locksmith/src/controllers/v2/ticketsController.ts

```diff
@@ -108,11 +109,15 @@ export class TicketsController {
         keyId,
         network
       )
+
+      const lock: Lock = await this.web3Service.getLock(lockAddress, network)
```

Comment:
@searchableguy we have a more efficient way to get it? just need to get the lockName without passing it as a parameter